### ISSUE TITLE
NVDA and Word viewer

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2399]
**Reported by challsworth2 on 2012-05-30 11:23**
NVDA does not read the content of documents opened in Microsoft Word Viewer. This applies for both standard and object navigation.

**Comment by bartekbrak** [comment]
Word Viewer is the only program I found so far to view files in read only mode which is essential for the machine I'm administering for my blind uncle. Any other editors will ask if the file is to be saved if alphanumeric key is pressed while viewing displaying a dialog box which confuses the user. 

The link to the program is here: https://www.microsoft.com/en-US/download/details.aspx?id=4

I confirm the problem on Windows 7. nvda 2013.3. Will check the newest version now.

**Comment by jcsteh** [comment]
If I recall correctly, Word Viewer doesn't include/allow access to the API which NVDA needs in order to access document content. Thus, there is no way we can make this accessible. @MichaelDCurran, is this definitely the case or does this need re-testing?

**Comment by michaelDCurran** [comment]
Never hurts to test again.

Certainly something like createObject word.application would never work, 
but I don't think we do that kind of thing anymore.



On 15/02/2016 8:06 AM, James Teh wrote:
>
> If I recall correctly, Word Viewer doesn't include/allow access to the 
> API which NVDA needs in order to access document content. Thus, there 
> is no way we can make this accessible. @MichaelDCurran 
> <https://github.com/MichaelDCurran>, is this definitely the case or 
> does this need re-testing?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/nvaccess/nvda/issues/2399#issuecomment-183990020>.
>

-- 
Michael Curran
Executive Director, NV Access Limited
Phone: +61 7 3149 3306
Website: http://www.nvaccess.org/
Twitter: @NVAccess
Facebook: http://www.facebook.com/NVAccess

**Comment by ehollig** [comment]
According to [Wikipedia](https://en.wikipedia.org/wiki/Microsoft_Word_Viewer), Word Viewer is discontinued and download will be unavailable in November 2017.

**Comment by ehollig** [comment]
Closing because of reasoning in https://github.com/nvaccess/nvda/issues/2399#issuecomment-319172854
